test/unit/utils/jwt.spec.cjs

```diff
@@ -184,4 +184,69 @@ describe('Utils::JWT', () => {
       expect(decoded1.exp).to.be.closeTo(decoded2.exp, 2);
     });
   });
+
+  describe('signViewerCounts', () => {
+    /** @test {Mux.Utils.JWT.signViewerCounts} */
+    it('defaults to rt and includes an expiration', () => {
+      const options = {
+        type: 'video',
+        keyId: TEST_ID,
+        keySecret: TEST_SECRET,
+      };
+
+      const token = JWT.signViewerCounts('some-space-id', options);
```

Comment:
It's only a test so it doesn't matter, but should it be `some-playback-id`?

Reply:
It's actually a video_id (because we set `type: 'video'` in the `options`), and I've updated it.